src/components/input/input.html

```diff
@@ -12,11 +12,20 @@
              [attr.aria-disabled]="ariaDisabled"
              [attr.aria-required]="ariaRequired"
              [attr.aria-invalid]="ariaInvalid"
-             [id]="id"
+             [attr.autocomplete]="autoComplete"
```

Comment:
All of the bindings you're adding in this PR are actual properties on `HTMLInputElement`, so none of them need `attr.`